assemblyline/odm/models/ontology/results/process.py

```diff
@@ -7,19 +7,21 @@
 
 @odm.model(description="Details about the characteristics used to identify an object")
 class ObjectID(odm.Model):
-    guid = odm.Optional(odm.Text(), description="The GUID associated with the object")
     tag = odm.Text(description="The normalized tag of the object")
+    ontology_id = odm.Keyword(description="Unique identifier of ontology")
```

Comment:
Maybe a more apt descriptor like we talked about? Since the ontology_id is more deterministic than unique/random.

Reply:
https://github.com/CybercentreCanada/assemblyline-base/pull/853/commits/cb788f1581b4a140e8152d3e466d9446c35ee123